storagenode/piecestore/verification_test.go

```diff
@@ -322,10 +322,12 @@ func setSpace(ctx context.Context, t *testing.T, planet *testplanet.Planet, spac
 		availableSpace, err := storageNode.Storage2.Monitor.AvailableSpace(ctx)
 		require.NoError(t, err)
 		diff := (space - availableSpace) * -1
+		now := time.Now()
```

Comment:
do we need to use `time.Now().UTC()`?

Reply:
Why should we?

Reply:
I thought `time.Now()` used the local timezone, while the dates used for the bloom filters sent from the satellite will be UTC